roles/openshift_logging_curator/templates/curator-cj.j2

```diff
@@ -0,0 +1,113 @@
+apiVersion: "batch/v1beta1"
+kind: "CronJob"
+metadata:
+  name: "{{deploy_name}}"
+  labels:
+    provider: openshift
+    component: "{{component}}"
+    logging-infra: "{{logging_component}}"
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  schedule: "{{ cron_job_schedule }}"
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      concurrencyPolicy: Forbid
+      replicas: {{curator_replicas|default(1)}}
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          name: "{{deploy_name}}"
+          labels:
+            logging-infra: "{{logging_component}}"
+            provider: openshift
+            component: "{{component}}"
+        spec:
+          restartPolicy: Never
+          terminationGracePeriod: 600
+          serviceAccountName: aggregated-logging-curator
+{% if curator_node_selector is iterable and curator_node_selector | length > 0 %}
+          nodeSelector:
+{% for key, value in curator_node_selector.iteritems() %}
+            {{key}}: "{{value}}"
+{% endfor %}
+{% endif %}
+          containers:
+            -
+              name: "curator"
+              image: {{image}}
+              imagePullPolicy: IfNotPresent
+{% if (curator_memory_limit is defined and curator_memory_limit is not none and curator_memory_limit != "") or (curator_cpu_limit is defined and curator_cpu_limit is not none and curator_cpu_limit != "") or (curator_cpu_request is defined and curator_cpu_request is not none and curator_cpu_request != "") %}
```

Comment:
@ewolinetz do we have patterns we are using else where to maybe make this less ugly?

Reply:
Can we use the fact that its python and just have something like `curator_memory_limit is defined and curator_memory_limit` ?